rpc_jobs/rpc_ceph.yml

```diff
@@ -57,6 +100,7 @@
 
     scenario:
       - "functional"
+      - "functional_mimic"
```

Comment:
Any reason bluestore_mimic isn't being added here?

Reply:
Having bluestore_mimic here would provide value.  I left it off to reduce the number of gates we are running to conserve testing resources.  I was worried that we may have too many gates and was being conservative for now.  But I am happy to add it in and we can always come back and modify it later if it becomes a problem.